x-pack/auditbeat/module/system/login/config.go

```diff
@@ -0,0 +1,19 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package login
+
+// Config defines the metricset's configuration options.
+type Config struct {
+	UtmpFilePattern string `config:"login.utmp_file_pattern"`
+}
+
+// Validate validates the login metricset config.
+func (c *Config) Validate() error {
+	return nil
+}
+
+var defaultConfig = Config{
```

Comment:
Note: As configs tend to get more complex newer code should better use functions to prepare an initial config object. This reduces the risk of accidentily sharing configs if references (e.g. map or pointers) are used in the defaults.

This is no issue here, but some recommendation to keepin in mind in general.